docs/machine-learning/README.md

```diff
@@ -19,7 +19,8 @@ The diagram below shows different phases in an ideal ML project. Due to practica
 
 ![Project flow](images/flow.png)
 
-* **[Envisioning](ml-problem-formulation-envisioning.md)**: Initial problem understanding, customer goals and objectives.
+* **[Envisioning](ml-problem-formulation-envisioning.md)**: Initial problem understanding, customer goals and objectives. See [template for envisioning outcomes summary document](ml-envisioning-summary-template.md).
+* **[Data Exploration](ml-data-exploration.md)**: Initial in-depth assessment of the data.
```

Comment:
It might not be clear to the reader how are data exploration and feasibility study related. Consider putting the link to Data Exploration in the "general guidance" list below.